### PR TITLE
Update import to fix a deprecation warning

### LINKED ISSUE #1129: Deprecation warning importing UnrestrictedUser
```
/home/ale/Code/plone/projects/coredev/6.0/src/plone.restapi/src/plone/restapi/services/workflow/utils.py:6: DeprecationWarning: UnrestrictedUser is deprecated. The functionality of AccessControl.User has moved to AccessControl.users. Please import from there. This backward compatibility shim will be removed in AccessControl version 6.
  from AccessControl.User import UnrestrictedUser as BaseUnrestrictedUser
```

### PR DESCRIPTION
**Pull request by ale-rt**
Fixes #1129

### CODE REVIEW

**Comment by ale-rt** [comment]
@tisto if that is interesting I can backport this patch to the 7.x branch in the form:
```
try:
    from AccessControl.users import UnrestrictedUser as BaseUnrestrictedUser
except ImportError:
    from AccessControl.User import UnrestrictedUser as BaseUnrestrictedUser
```

File: news/1129.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix a deprecation warning
```
Comment (tisto) [inline news/1129.bugfix:1]: @ale-rt maybe we can be a bit more specific here :)

**Comment by tisto** [comment]
@ale-rt yeah. A backport would be nice. Let's get that one in first though.

File: news/1129.bugfix (hunk 1 above)
Comment (ale-rt) [reply news/1129.bugfix]: Is it fine now?

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]

**Pull request merged**